src/index.test.ts

```diff
@@ -1,11 +1,72 @@
-import * as strongConfig from './index'
+jest.mock('./load')
+jest.mock('./validate')
 
-describe('strong-config exports are as expected', () => {
-  it('exports a load function', () => {
-    expect(strongConfig.load).toBeInstanceOf(Function)
+import { load } from './load'
+import { validate } from './validate'
+
+const mockedLoad = load as jest.MockedFunction<typeof load>
+const mockedValidate = validate as jest.MockedFunction<typeof validate>
+
+const mockedConfig = { some: 'config', runtimeEnvironment: 'development' }
+const mockedValidationResult = true
+mockedLoad.mockReturnValue(mockedConfig)
+mockedValidate.mockReturnValue(mockedValidationResult)
+
+import StrongConfig from './index'
+
+describe('StrongConfig class', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('StrongConfig is a class that can be instantiated', () => {
+    expect(new StrongConfig()).toBeDefined()
   })
 
-  it('exports a validate function', () => {
-    expect(strongConfig.validate).toBeInstanceOf(Function)
+  it('StrongConfig accepts and stores a parameter object', () => {
+    const mockedParams = {}
+    const strongConfig = new StrongConfig(mockedParams)
+
+    expect(strongConfig.params).toStrictEqual(mockedParams)
+  })
+
+  it('strongConfig exposes "load()"', () => {
+    expect(new StrongConfig().load).toBeInstanceOf(Function)
+  })
+
+  it('strongConfig exposes "validate()"', () => {
+    expect(new StrongConfig().validate).toBeInstanceOf(Function)
+  })
+
```

Comment:
here you're beginning a new context, so you should wrap the below in a new `describe()`

```suggestion
describe('load()', () => {
```

Reply:
updated